app/views/admin/api_docs/base/_form.html.slim

```diff
@@ -9,7 +9,8 @@
     = form.input :system_name, input_html: { :disabled => true }
   = form.input :published, label: 'Publish?'
   = form.input :description, input_html: {rows: 3}
-  = form.input :service_id, as: :select, collection: @service_apis
+  - if !@api_docs_service.new_record? || @api_docs_service.service.blank?
```

Comment:
Can't we change the service of an api_doc?

Reply:
Yup 😉 
![image](https://user-images.githubusercontent.com/11318903/46818843-74927d00-cd82-11e8-9335-2c4a390ce5f6.png)